### PR TITLE
registration-setup

### PR DESCRIPTION
**Pull request by alisarfaraj**

### CODE REVIEW

File: src/pages/setup-profile/index.tsx [hunk 1]
```diff
@@ -70,22 +55,22 @@ function ProfileSetup() {
     } else {
       dispatch(setPreLoader(true));
       const payload = {
-        userTypeId: roleId,
-        userTypeName: role,
+        user_roles: role,
+        default_profile: role,
```
Comment (muddledluck) [inline src/pages/setup-profile/index.tsx:59]: use `camelCase`

File: src/redux/reducers/userRole.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { USER_ROLES } from "@/utils/enum";
 
 // Define a type for the slice state
 interface userRoleI {
-  userRole: USER_ROLES | null
+  userRole: USER_ROLES | null;
```
Comment (muddledluck) [inline src/redux/reducers/userRole.ts:8]: if no need then remove this file

File: src/redux/store/store.ts [hunk 3]
```diff
@@ -2,16 +2,18 @@ import { configureStore } from "@reduxjs/toolkit";
...
 import ModalsToogleReducer from "../reducers/modalsToggle";
+import userReducer from "../reducers/user";
 
 export const store = configureStore({
   reducer: {
     UserRoleReducer,
     PreLoaderReducer,
-    ModalsToogleReducer
+    ModalsToogleReducer,
+    userReducer,
```
Comment (muddledluck) [inline src/redux/store/store.ts:12]: follow same name conventations

File: src/utils/interfaces.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import { USER_ROLES } from "./enum";
+
+export interface NewIUserType {
+  id: string;
+  user_roles: USER_ROLES;
```
Comment (muddledluck) [inline src/utils/interfaces.ts:5]: use `camelCase`

**Review by muddledluck — CHANGES_REQUESTED** [review]

File: src/pages/setup-profile/index.tsx (hunk 1 above)
Comment (0422-shiv) [reply src/pages/setup-profile/index.tsx:59]: It's backend fields so no need to change camel case

File: src/redux/reducers/userRole.ts (hunk 2 above)
Comment (0422-shiv) [reply src/redux/reducers/userRole.ts:8]: No need to remove this file

File: src/component/lightBoxes/imageCropper.tsx [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+      );
+
+      setCroppedImage(croppedImage);
+      setTimeout(() => {
+        dispatch(handleImageCropperToggle(false));
+      }, 1000);
+    } catch (e) {
+      console.error(e);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
[the remark this answers is not in the document]
Comment (alisarfaraj) [reply src/component/lightBoxes/imageCropper.tsx:56]: couldn't remove it

**Pull request merged**